components/account/UpdatePassword.tsx

```diff
@@ -8,8 +8,8 @@ import { Card, InputWithLabel } from '@/components/shared';
 import { defaultHeaders } from '@/lib/common';
 
 const schema = Yup.object().shape({
-  currentPassword: Yup.string().required(),
-  newPassword: Yup.string().required().min(7),
+  currentPassword: Yup.string().required().min(8),
```

Comment:
```suggestion
  currentPassword: Yup.string().required(),
```
We can't force min length check on current password.